smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -191,6 +191,18 @@ def has_output_keys(self) -> bool:
         return self.output_keys is not None and bool(self.output_keys)
 
 
+@dataclass
+class TensorMeta:
+    """Metadata about a tensor, built from TensorDescriptors."""
+
+    dimensions: t.List[int]
+    """Dimensions of the tensor"""
+    order: str
+    """Order of the tensor"""
```

Comment:
Consider being more specific here:
`In-memory order of the tensor. E.g. supply C or R to indicate column or row orientation`

Reply:
@al-rigazzi what is a good, backend-agnostic description?

Reply:
added more description